### ISSUE TITLE
Expose environment variables from config file

### ISSUE THREAD

**Issue by idantene** [allegroai/clearml-agent#126]
Hey!

Just a suggestion for the `agent` configuration part. I think it would be beneficial if the user could specify e.g. that the definitions in `clearml.conf` under `sdk.s3` (or similar) or exposed as environment variables for the running task.
That way, `clearml.conf` can be the source of truth and defines e.g. `AWS_ACCESS_KEY` when running tasks.

I see the support is already there (in `EnvironmentConfig.set` and friends), but it's only directly called (I believe) for `CLEARML_`-prefixed variables. Therefore, it would be nice if the user could specify a boolean or a list of variables to expose like this.

Thanks!

**Comment by idantene** [comment]
Found the following ad-hoc solution, though maybe this can still be supported directly?

Apparently we can also use the `environment` section in `clearml.conf`. Then, one can use HOCON's self-refential to access the values:
```hocon
environment {
  AWS_ACCESS_KEY_ID: ...
}
sdk.aws.s3.key: ${environment.AWS_ACCESS_KEY_ID}
```

**Comment by jkhenning** [comment]
Yeah, I think this is actually quite a neat solution that also uses the HOCON capabilities (good thing we use it 🙂)
